src/store/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const SET_DATA: string = 'SET_DATA';
```

Comment:
Please consider separating this into two files.
I'd suggest to create `actions-type.ts` with all the action types you would have.
And the API_URL is something related to app configuration, so it should be placed in a proper place. (hint: not in src/store/)

Reply:
Done